streaming/base/format/json/writer.py

```diff
@@ -11,6 +11,19 @@
 
 
 class JSONWriter(SplitWriter):
+    """Writes a streaming JSON dataset.
+
+    Args:
+        dirname (str): Local dataset directory.
```

Comment:
Did you compile and ran the doc locally to see if it renders properly?

Reply:
https://mosaicml-composer--14.com.readthedocs.build/projects/streaming/en/14/